cachito/web/static/api_v1.yaml

```diff
@@ -34,6 +34,13 @@ paths:
             type: boolean
             example: true
             default: false
+        - name: state
```

Comment:
Could you put this above `verbose`? It's nice to have the documentation in alphabetical order for readability.